docs/source/use_cases/analyze_variants/gwas.rst

```diff
@@ -0,0 +1,40 @@
+Genome-Wide Association Study (GWAS)
+====================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/analyze_variants/gwas.html
+
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+`Google BigQuery`_ can be used to perform a GWAS.  Here are several examples:
+
+* Chi-squared tests on :doc:`/use_cases/discover_public_data/1000_genomes` dataset with members of EAS super population as case and control all other populations:
+
+  * iPythonNotebook `Genome-wide association study (GWAS).ipynb <https://github.com/googlegenomics/datalab-examples/blob/master/datalab/genomics/Genome-wide%20association%20study%20(GWAS).ipynb>`_
+  * SQL `gwas-pattern-chi-squared-test.sql <https://github.com/googlegenomics/bigquery-examples/blob/master/1000genomes/sql/gwas-pattern-chi-squared-test.sql>`_
+
+* Two-proportion Z test on :doc:`/use_cases/discover_public_data/1000_genomes` dataset with members of EAS super population as case and control all other populations:
+
+  * SQL `gwas-pattern-two-proportion-z-test.sql <https://github.com/googlegenomics/bigquery-examples/blob/master/1000genomes/sql/gwas-pattern-two-proportion-z-test.sql>`_
```

Comment:
In the SQL it says:

```
# z-score critical value for p-value=5*10^-8 is +/-5.326724`
# > qnorm(1 - 5e-08)
# [1] 5.326724
```
But this is a two-sided test, so the p-value would need to be divided in half:

```
> qnorm(1 - 5e-08/2)
[1] 5.45131
```
So the following would need to be updated in the query:
```
z_score >= 5.326724
  OR z_score <= -5.326724
```